countryinfo.html

```diff
@@ -24,54 +24,79 @@
   <script src="./assets/js/countryinfo.js" defer></script>
 </head>
 
-<body style="background-image: none; opacity:0.5;">
+<body style="background-color: darkgray;">
 
   <!-- Main container for page content -->
   <div class="container">
     <!-- Progress bar -->
     <div class="progress" id="progressBar" style="display:none">
       <div class="indeterminate"></div>
+      <h1 id="progressTxt"></h1>
     </div>
+    
     <header class="YourSearch">
-
     </header>
-
-    <!-- Country info main section -->
-    <div class="row cards">
-      <div class="col s4 offset-s1 card" id="countryInformation"> </div>
-      <div class="col s4 offset-s1 card" id="InfoList">
-        <!-- <div id="emoji"></div> -->
+    
+
+    <div class="row col s12">
```

Comment:
What does this do?

Reply:
This is to create two rows of cards, 1st row is the original row, line 65 is the new and 2nd row of cards, which will be used to display search card and search history.

Reply:
Hi Ray - Some good changes (make sure to remove any unused console.logs